packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -182,7 +188,9 @@ async function normalizeArgsMiddleware(
         if (monorepoStyle === 'package-based') {
           preset = 'npm';
         } else if (monorepoStyle === 'react') {
-          preset = Preset.ReactStandalone;
+          preset = (await isFrameworkNextJs(argv))
```

Comment:
Can we rename this to `determineReactFramework` and return the preset value from the function. So it can be just:

```ts
preset = await determineReactFramework(argv);
```